docstr_coverage/cli.py

```diff
@@ -261,6 +261,24 @@ def _assert_valid_key_value(k, v):
     default=".docstr_coverage",
     help="Deprecated. Use json config (--config / -C) instead",
 )
+@click.option(
+    "-o",
+    "--output",
+    type=click.Choice(["stdout", "file"]),
+    default="stdout",
+    help="Format of output",
```

Comment:
This isn't really the "format" of the output (that comes from the `--format` option. Maybe output target/location/destination. I can't really think of the right word. But this option is used to specify "where" to save the output rather than how to format the output

Reply:
You right. I think the `destination` with combination of choices (`stdout`, `file`) clearly saying about param appointment. 

Maby change `--output` to `--destination`?

Reply:
Yeah, that's a good idea. I like `--destination`